src/index.js

```diff
@@ -0,0 +1,233 @@
+        cityNameHeader.textContent = event.target.value;
+    });
+}
+
+// updates temperature text color and weather garden landscape
+const updateWeatherDisplay = () => {
+    let temp = kelvinToF(currentTemperature.value);
+    if (temp >= 80) {
+        tempValue.style.color = "red";
+        landscape.textContent = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+
+    } else if (temp >= 70) {
+        tempValue.style.color = "orange";
+        landscape.textContent = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+
+    } else if (temp >= 60) {
+        tempValue.style.color = "yellow";
+        landscape.textContent = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+
+    } else if (temp >= 50) {
+        tempValue.style.color = "green";
+        landscape.textContent = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+    } else {
+        tempValue.style.color = "teal";
+        landscape.textContent = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+    }
+    tempValue.textContent = displayTemperature();
+    updateBackground();
+};
+
+// weather album section
+const updateWeatherAlbum = () => {
+    const images = [
+        './assets/weather-imgs/weather_1.png',
+        './assets/weather-imgs/weather_2.jpg',
+        './assets/weather-imgs/weather_3.png',
+        './assets/weather-imgs/weather_4.jpg',
+        './assets/weather-imgs/weather_5.jpg'
+    ];
+    let currentIndex = 0;
+
+    // rotates weather album images
+    const rotateImage = () => {
+        currentIndex = (currentIndex + 1) % images.length;
+        landscapeImg.style.backgroundImage = `url(${images[currentIndex]})`;
+    };
+
+    setInterval(rotateImage, 10000);
+};
+
+// reset city name section
+const updateCityNameReset = () => {
+    // resets city name and temperature
+    const resetCityInfo = () => {
+        cityNameHeader.textContent = defaultCityNameInput;
+        currentTemperature.value = defaultTempValue
+        tempValue.textContent = displayTemperature();
+        tempValue.style.color = "red";
+        cityNameTextBox.value = "";
+        updateWeatherDisplay();
+        updateBackground();
+    };
+
+    resetCityName.addEventListener("click", resetCityInfo);
+};
+
+// Get Temp of search terms
+const tempSearch = async (cityName) => {
+    const locationResponse = await axios.get(`${baseURL}/location`, {
+        params: {q: cityName}
+    });
+    const {lat, lon} = locationResponse.data[0];
+
+    const weatherResponse = await axios.get(`${baseURL}/weather`, {
+        params: {lat, lon}
+    });
+    currentTemperature.value = weatherResponse.data.main.temp;
+    tempValue.textContent = displayTemperature();
+    updateWeatherDisplay();
+};
+
+// - Changing the temperature should change the page's background.
+const updateBackground = () => {
+    const body = document.body;
+    let temp = kelvinToF(currentTemperature.value);
+    if (temp > 80) {
+        currentTemperature.image = "hot.png"
+    } else if (temp > 65) {
+        currentTemperature.image = "good.png"
+    } else {
+        currentTemperature.image = "cold.png"
+    }
```

Comment:
Consider using a data structure here to hold the info related to which image should be displayed. Then you can use `temp` to determine which image to display

```js
images = {
  80: "hot.png",
  64: "good.png",
  // etc...
}
```